storage/id_alloc.go

```diff
@@ -114,6 +114,7 @@ func (ia *IDAllocator) allocateBlock(incr int64) {
 		}
 		idKey := ia.idKey.Load().(proto.Key)
 		if err := ia.db.Call(proto.Increment, proto.IncrementArgs(idKey, incr), ir); err != nil {
+			log.Warningf("unable to allocate %d ids from %q: %s", incr, ia.idKey, err)
```

Comment:
`s/ia.idKey/idKey/g` and use `%s` instead of `%q` for formatting the key (though that doesn't actually matter anymore).